File: backend/services/assessment-service/src/main/java/com/lmspilot/assessment/AssessmentServiceApplication.java
package com.lmspilot.assessment;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;


@SpringBootApplication(scanBasePackages = "com.lmspilot")
public class AssessmentServiceApplication {
    public static void main(String[] args) { SpringApplication.run(AssessmentServiceApplication.class, args); }
}

File: backend/services/assessment-service/src/main/kotlin/com/lmspilot/assessment/AssessmentServiceApplication.kt
package com.lmspilot.assessment

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@SpringBootApplication(scanBasePackages = ["com.lmspilot"])
class AssessmentServiceApplication

fun main(args: Array<String>) {
    runApplication<AssessmentServiceApplication>(*args)
}
